usr/src/uts/armv8/os/gic.c

```diff
@@ -184,9 +184,9 @@ gic_addspl(int irq, int ipl, int min_ipl, int max_ipl)
 static int
 gic_delspl(int irq, int ipl, int min_ipl, int max_ipl)
 {
-	ASSERT3S(irq, <, MAX_VECT);
+	int pri = 42;
```

Comment:
I don't understand

Reply:
This is a terrible habit I've picked up - a Douglas Adams reference :)

Basically, the number doesn't matter as long as it's not what I'm checking for later. I'll update it to something else.